src/ZendCodingStandard/Sniffs/Commenting/FileLevelDocBlockSniff.php

```diff
@@ -0,0 +1,261 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-coding-standard for the canonical source repository
+ * @copyright https://github.com/zendframework/zend-coding-standard/blob/master/COPYING.md Copyright
+ * @license   https://github.com/zendframework/zend-coding-standard/blob/master/LICENSE.md New BSD License
+ */
+
+use PHP_CodeSniffer_File as File;
+use PHP_CodeSniffer_Sniff as Sniff;
+use Zend\CodingStandard\Utils\LicenseUtils;
+
+/**
+ * FileLevelDocBlock Sniff
+ *
+ * - Checks if a file has a valid file-level docblock
+ * - Checks for missing/invalid see tag
+ * - Checks for missing/invalid copyright tag
+ * - Checks for missing/invalid license tag
+ * - Checks order of see, copyright and license tags
+ */
+class ZendCodingStandard_Sniffs_Commenting_FileLevelDocBlockSniff implements Sniff
+{
+    /**
+     * @var string
+     */
+    private $repo;
+
+    const IGNORE = [
+        T_CLASS,
+        T_INTERFACE,
+        T_TRAIT,
+        T_FUNCTION,
+        T_CLOSURE,
+        T_PUBLIC,
+        T_PRIVATE,
+        T_PROTECTED,
+        T_FINAL,
+        T_STATIC,
+        T_ABSTRACT,
+        T_CONST,
+        T_PROPERTY,
+        T_INCLUDE,
+        T_INCLUDE_ONCE,
+        T_REQUIRE,
+        T_REQUIRE_ONCE,
+    ];
+
+    public function __construct()
+    {
+        // Get current repo name from composer.json
+        $content = file_get_contents('composer.json');
```

Comment:
If getpwd() returns the directory from where phpcs/phpcbf is executed than that is even better.